src/index.ts

```diff
@@ -21,6 +22,13 @@ export const VerifiedSnapStruct = object({
   id: string(),
   metadata: object({
     name: string(),
+    type: optional(enums(['account'])),
+    author: optional(string()),
+    website: optional(string()),
+    summary: optional(string()),
```

Comment:
What's the difference between a summary and a description here?

Reply:
`summary` is a short summary that we will display in the cards view, `description` is a (longer) description that we show when a snap is selected from the list.

It's the same meaning as the in the [OpenAPI specs](https://spec.openapis.org/oas/v3.1.0#fixed-fields-0).